compair/tests/api/test_comparisons.py

```diff
@@ -317,12 +325,16 @@ def test_get_and_submit_comparison(self, mocked_update_assignment_grades_run, mo
                     actual_comparison = rv.json['comparison']
                     self._validate_comparison_submit(comparison_submit, actual_comparison, expected_comparison)
 
-                    # grades should increase for every comparison
+                    # grades should increase for every comparison, unless student has done more than required by assignment
                     if user.id == self.data.get_authorized_student().id:
                         new_course_grade = CourseGrade.get_user_course_grade(self.course, user)
                         new_assignment_grade = AssignmentGrade.get_user_assignment_grade(self.assignment, user)
-                        self.assertGreater(new_course_grade.grade, course_grade)
```

Comment:
due to code change you shouldn't need the if anymore (will only go up to max times)